### DISCUSSION TITLE
VPS or Shared host 5000 listeners

### DISCUSSION THREAD

**Discussion by KFarea** [AzuraCast/AzuraCast#6759] [Q&A]
VPS or shared host for Azuracast internet radio with almost 5000 listeners at the same time?
And if you have any recommendations for server specification, it will be great.

Thanks.

**Comment by Heimatradio** [comment]
VPS, enough RAM & CPU's, and the most important thing: at least high internet connection
have a look here https://www.internet-radio.com/servers/tools/bandwidth/

**Comment by Moonbase59** [comment]
For pure audio streams (but video ain’t bad either!), Icecast is _incredibly performant_ here. I’ve been testing up to 5000 streams on a 2 CPU, 4 GB VM a while back, and it worked without issues (on a Digital Ocean instance). So it’s mainly ensuring you get enough (and constant) bandwidth. With that many real listeners, you might of course want to think about some failover and load balancing.

**Reply by druff99** [reply]
"failover and load balancing. - do you have any recommendation for such ?
